### DISCUSSION TITLE
What is a Git repository?

### DISCUSSION THREAD

**Discussion by montasim** [STRINGLABITSOLUTIONS/Bhojon#235] [Q&A] [answered]
Short answer

**Comment by SamsonGoodenough** [comment, marked as answer]
A Git repository is like the home folder for a developer's project, it stores all the project's files and changes made to them (this is called [source control](https://aws.amazon.com/devops/source-control/)). Repositories allow users to interact with other people's work, suggest edits, inform the author of issues, fork it for your own use, etc... Repos also allow the author to change the access permissions so only certain people can view or edit code, making them a vital tool for collaboration.

**Reply by montasim** [reply]
@SamsonGoodenough you need two accepted answers to get a badge. Try answering other questions.

By the way, would you mind giving me a star https://github.com/montasim/montasim

**Comment by fcasamento** [comment]
A Git repository is virtual storage for projects. It allows you to save versions of the code, which you can access when you need to. It can be remote or local.
